packages/testsuite/cypress/support/navigation.ts

```diff
@@ -43,6 +43,13 @@ Cypress.Commands.add("navigateToUpdateManagerPage", (managementEndpoint, address
   cy.get("#hal-root-container").should("be.visible");
 });
 
+Cypress.Commands.add("logoutFromWebConsole", () => {
+  cy.get(
+    `#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility > li:nth-child(5) > a`
```

Comment:
> `#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility > li:nth-child(5) > a`

What about `#header-username` instead?

Or, in case it's not sufficient, what about `cy.get("#header-username").parent().click()`?

Reply:
No, The `#header-username` selector cannot be used. The shortest selector is `#hal-header-container > div.collapse.navbar-collapse.hal-header-collapse > ul.nav.navbar-nav.navbar-utility`

Reply:
And the `cy.get("#header-username").parent()` variant? Does it return a different object? What about something like `.parents("a")` instead?

Reply:
I didn't know about parents method. Thank you. Updated.